<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoFileConnectorDialogForm</class>
 <widget class="QDialog" name="GeoFileConnectorDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoFile Connector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0,3,1,0,0">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">QWidget { background: white }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="m_datasourceTypeTitleLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GeoFile - Common raster and vector file formats</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Data Source List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="m_datasourceListWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Data Source</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="m_datasourceLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="m_fileSearchToolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_datasourceTitleLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="m_datasourceDescriptionTextEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolTip">
             <string>A more detailed description about the data source</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="m_addDatasourceToolButton">
         <property name="toolTip">
          <string>add data source to the list</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_removeDatasourceToolButton">
         <property name="toolTip">
          <string>remove selected data source from the list</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_updateDatasourceToolButton">
         <property name="toolTip">
          <string>update selected data source information</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
     <item>
      <widget class="QPushButton" name="m_helpPushButton">
       <property name="text">
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="m_testPushButton">
         <property name="text">
          <string>&amp;Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_openPushButton">
         <property name="text">
          <string>&amp;Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_closePushButton">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_datasourceListWidget</tabstop>
  <tabstop>m_fileSearchToolButton</tabstop>
  <tabstop>m_datasourceLineEdit</tabstop>
  <tabstop>m_datasourceTitleLineEdit</tabstop>
  <tabstop>m_datasourceDescriptionTextEdit</tabstop>
  <tabstop>m_openPushButton</tabstop>
  <tabstop>m_closePushButton</tabstop>
  <tabstop>m_testPushButton</tabstop>
  <tabstop>m_addDatasourceToolButton</tabstop>
  <tabstop>m_removeDatasourceToolButton</tabstop>
  <tabstop>m_updateDatasourceToolButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_closePushButton</sender>
   <signal>pressed()</signal>
   <receiver>GeoFileConnectorDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
